src/organisms/forms/GeneralForm/Readme.md

```diff
@@ -7,22 +7,26 @@ _This is a form wrapper that can take components as children._
       onSubmit={ () => alert('FormSubmitted!') }
     >
       <TextComponent tag='p' type={ 3 } regular>First, a little about you and your humble abode</TextComponent>
+
       <TextField label='First name' placeholder='Enter your first name here' forProp=''/>
+
       <LinkWrapper href="#">
         <TextComponent>
           Want to add more people to the policy?
         </TextComponent>
       </LinkWrapper>
-      <SelectField
-        forProp=''
-        label='Home type'
-        placeholder='Select your home type'
+
+      <SimpleSelectField
+        label='Select Field Label'
+        placeholder='Placeholder'
         selectOptions={[
-        ['Value 1', 'Option 1'],
-        ['Value 2', 'Option 2'],
-        ['Value 3', 'Option 3']
+          { value: 'Option 1' },
+          { value: 'Option 2' },
+          { value: 'Option 3' }
```

Comment:
@drewdrewthis I think the `value` prop on the field itself should be the same name as the text it displays. this will make it much easier for end users to determine what option was actually selected. It's also a practice I've seen, particularly with handling `onChange` events with react